androidapp/HopSquadApp/app/src/main/java/com/hopsquad/hopsquadapp/activities/MainActivity.java

```diff
@@ -78,6 +80,7 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
     };
 
     private void switchFragment(Fragment fragment, String tag) {
+        currentFragment = fragment;
```

Comment:
I think we also need to update the name of the view right? it currently shows Main Activity for all. Would this be something that gets done here or in the fragment directly?

Reply:
This would be done on the MainActivity layout. Maybe even the AndroidManifest